### PR TITLE
Update build-cap-app.md

### PR DESCRIPTION
**Pull request by hm23**
I found a few broken links and outdated/unformatted snippets, please check.

In step 1, you only mention the windows/linux keyboard shortcut for opening the BAS Explorer. I propose to also mention the Mac variant. 

In step 2, the `schema.cds` snippet contains an unused import `User`. I removed it (I hope it's not necessary at some later point in the tutorial)

In step 3, the `index.html` looks different ⚠️ **with cds8** ⚠️ :

index

... but I don't know in which quality you want to have the screenshot, so I didn't add it to the PR.

### CODE REVIEW

File: tutorials/build-cap-app/build-cap-app.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ You have the SAP Business Application Studio configured. See [Set Up SAP Busines
 
     <!-- border; size:540px --> ![Open SAP Business Application Studio folder](./bas-open-folder.png)
 
-    > You can also use the **Ctrl+Shift+E** key combination to quickly navigate to the **Explorer**.
+    > You can also use the **Cmd+Shift+E** (Mac) or **Ctrl+Shift+E** (Windows/Linux) key combination to quickly navigate to the **Explorer**.
```
Comment (hm23) [inline tutorials/build-cap-app/build-cap-app.md:55]:
```suggestion
    > You can also use the **Cmd+Shift+E** (macOS) or **Ctrl+Shift+E** (Windows/Linux) key combination to quickly navigate to the **Explorer**.
```

**Comment by hm23** [comment]
Hi @slavipande,

Thanks for looking into this. I just noticed that the new look of the `index.html` comes with cds8. I followed your tutorial on a canary BAS, which already uses cds8. The BAS of my trial account however uses cds7, so your current `index.html` screenshot is not (yet) outdated.

As far as I can see, the other proposed changes are valid also when following the tutorial on trial.

**Review by slavipande — CHANGES_REQUESTED** [review]
@hm23,

Indeed, the new `index.html` comes with cds8 so for the time being we're not updating the screenshot. According to my dev colleagues, the change `[cds] - connect to db > sqlite { database: ':memory:' }` --> `[cds] - connect to db > sqlite { url: ':memory:' }` also comes with cds8, so this will also have to be postponed for now. 

Could you please revert this part of the suggested changes and I'll be happy to merge the rest. 

Thanks and regards,
Sveto

**Comment by hm23** [comment]
@slavipande, I reverted the cds8 log change and the indentation of the cds snippets. Please note that I still propose to remove the User import in `schema.cds`, which is not used anywhere. I also propose to use CDS as language identifier for the cds snippets.

Another suggestion would be to remove the AdminService in `services.cds`, which is not used anywhere in this tutorial group, if I'm not mistaken.

I understand that you want to keep your tutorial content consistent. If I have feedback regarding the underlying incident app code, would it make more sense to open an issue/PR targeting the `beginner-tutorials` branch of https://github.com/cap-js/incidents-app?

Best regards,
Heiko (temporarily part of the CAP Enablement Team)

**Review by slavipande — APPROVED** [review]

**Comment by slavipande** [comment]
> @slavipande, I reverted the cds8 log change and the indentation of the cds snippets. Please note that I still propose to remove the User import in `schema.cds`, which is not used anywhere. I also propose to use CDS as language identifier for the cds snippets.
> 
> Another suggestion would be to remove the AdminService in `services.cds`, which is not used anywhere in this tutorial group, if I'm not mistaken.
> 
> I understand that you want to keep your tutorial content consistent. If I have feedback regarding the underlying incident app code, would it make more sense to open an issue/PR targeting the `beginner-tutorials` branch of https://github.com/cap-js/incidents-app?
> 
> Best regards, Heiko (temporarily part of the CAP Enablement Team)

@hm23 , thanks for updating your PR and also thanks for the additional suggestions!

I'll be merging your changes right away.

On your other suggestions:
- with regard to `AdminService`, this will be used in advanced tutorials that use this one as a prerequisite so we'd like to keep it as it is.
- in future, we'll be using CDS as a language identifier for all `.cds` files. Thanks for your suggestion!
- for feedback regarding the underlying incident app code, please get in contact or open an issue/PR targeting the `beginner-tutorials` branch of https://github.com/cap-js/incidents-app.

Best regards,
Sveto

**Pull request merged**